build/template-restore-build-libsandsamples.yaml

```diff
@@ -7,6 +7,14 @@ parameters:
   MsalClientSemVer: '4.0.0-devopsbuild'
 
 steps:
+
+- task: Bash@3
+  displayName: 'Enable NET 6 support'
+  inputs:
+    targetType: 'inline'
+    script: 'echo "##vso[task.setvariable variable=MAUI_SUPPORT;]true"'
+    condition: eq(variables['Net6_Support'], 'true')
```

Comment:
Added `Net6_Support` as a variable which is `true` in our classic builds. OneBranch has this as false